app/themes/config.js

```diff
@@ -467,6 +465,67 @@ export const ACTION_FIELDS = {
       optional: [ACTIONTYPES.DONOR],
       type: 'text',
     },
+    draft: {
+      defaultValue: true,
+      required: Object.values(ACTIONTYPES), // all types
+      type: 'bool',
+      skipImport: true,
+    },
+    private: {
```

Comment:
@vbojilova let's not add attributes that we don't have =) if in doubt, you can always refer to the db schema that you find here
https://github.com/dumparkltd/marine-defrag-server/blob/master/db/schema.rb#L234